extensions/security-jpa/runtime/src/main/java/io/quarkus/security/jpa/runtime/JpaSecurityUtil.java

```diff
@@ -0,0 +1,90 @@
+package io.quarkus.security.jpa.runtime;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Objects;
+
+import org.wildfly.security.password.PasswordFactory;
+import org.wildfly.security.password.WildFlyElytronPasswordProvider;
+import org.wildfly.security.password.interfaces.BCryptPassword;
+import org.wildfly.security.password.spec.EncryptablePasswordSpec;
+import org.wildfly.security.password.spec.IteratedSaltedPasswordAlgorithmSpec;
+import org.wildfly.security.password.util.ModularCrypt;
+
+/**
+ * Utility class used to produce bcrypt hashes using the Modular Crypt Format.
+ */
+public class JpaSecurityUtil {
```

Comment:
Based on #5113 and #5667 would it make sense to move an rename the class to something les "JPA related"? Like `io.quarkus.security.CryptoUtil` or similar?

Reply:
That's a good point, I'll try to move it.

Reply:
Given that it depends on Elytron and `quarkus-security` does not depend on it, I can move it to `quarkus-elytron-security-common` which ATM only holds the GraalVM substs that I need anyway.

I don't think it's worth adding it to ``quarkus-elytron-security` itself since it comes with baggage.

WDYT @stuartwdouglas that place would work for you?

Reply:
Works for me.

Reply:
Done. Had to move some things that init elytron/password and register them as service at runtime there, but I think it's reasonable: https://github.com/quarkusio/quarkus/pull/5660/commits/d80322935c72c81a61eb305d0de2013f3894501d